scripts/compile_coref_data.sh

```diff
@@ -1,46 +0,0 @@
-#!/bin/bash
-
-# This script downloads and compiles the Ontonotes 2012 data in a helpful format
```

Comment:
It compiled the Ontonotes dataset (you can download a skeleton format without the words, pos tags or parse trees, which you have to merge with the private LDC data) and then formatted all of the conll data into a single file, which is basically pretty unhelpful, because it removes all the domain information that the file structure gives.